pkg/minikube/reason/known_issues.go

```diff
@@ -997,6 +1006,32 @@ var guestIssues = []match{
 		},
 		Regexp: re(`'/var/lib/dpkg': No such file or directory`),
 	},
+	{
+		Kind: Kind{
+			ID:       "GUEST_INCORRECT_ARCH",
+			ExitCode: ExGuestUnsupported,
+			Advice:   "You might be using an amd64 version of minikube on a M1 Mac, use the arm64 version of minikube instead",
+			Issues:   []int{10243},
+		},
+		Regexp: re(`qemu: uncaught target signal 11 (Segmentation fault) - core dumped`),
+	},
+	{
+		Kind: Kind{
+			ID:       "GUEST_CERTS_EXPIRED",
+			ExitCode: ExGuestError,
+			Advice:   "Your minikube certs likely expired, as a workaround, clear your minikube home dir `minikube delete --all --purge`",
+			Issues:   []int{10948},
+		},
+		Regexp: re(`controlPlane never updated to`),
```

Comment:
this one seems to Loose Regex, any better more accurate regex?

Reply:
Improved